main.tf

```diff
@@ -1,11 +1,11 @@
 ## PROVIDER
 # Configure the OpenStack Provider
 provider "openstack" {
-  #user_name          = "" # use $OS_USERNAME
-  #tenant_name        = "" # use $OS_PROJECT_NAME
-  #tenant_id          = "" # use $OS_PROJECT_ID
-  #password           = "" # use $OS_PASSWORD
-  #auth_url           = "" # use $OS_AUTH_URL
+  user_name          = "anders.sorlin@yh.nackademin.se"
+  #tenant_name        = "anders"
+  tenant_id          = "5b9807a78af2489290b87c6680d226df"
+  password           = "Läroverksvägen29F" 
+  auth_url           = "https://ops.elastx.cloud:5000"
```

Comment:
run the rc file to set local env variables instead
`source nackademin-sorlin-openrc.sh`

the file can be downloaded from openstack
https://ops.elastx.cloud/project/api_access/openrc/